src/main.h

```diff
@@ -174,7 +177,8 @@ void UnregisterNodeSignals(CNodeSignals& nodeSignals);
  * @param[out]  dbp     If pblock is stored to disk (or already there), this will be set to its location.
  * @return True if state.IsValid()
  */
-bool ProcessNewBlock(CValidationState &state, CNode* pfrom, CBlock* pblock, bool fForceProcessing, CDiskBlockPos *dbp);
+bool ProcessNewBlock(CValidationState &state, CNode* pfrom, CBlock* pblock, bool fForceProcessing, CDiskBlockPos *dbp,
+libzcash::ProofVerifier& verifier = disabledVerifier);
```

Comment:
We should default to a new strict verifier, because otherwise it is possible to accidentally have no verification at all (as is currently the case in `LoadExternalBlockFile()`. Or just remove the default, and require the caller to specify the verifier.